### PR TITLE
Use the int64_t formatter name to allow strict format checking to wor…

### PR DESCRIPTION
**Pull request by schmidtw**
…k everywhere.

### CODE REVIEW

File: src/wrp-c.c [hunk 1]
```diff
@@ -1217,8 +1217,8 @@ static void decodeMapRequest( msgpack_object deserialized, struct req_res_t **de
 
             switch( ValueType.type ) {
                 case MSGPACK_OBJECT_POSITIVE_INTEGER: {
-                    printf( "Map value is int %ld\n", ValueType.via.i64 );
-                    sprintf( mapdecodeReq->metadata->data_items[v].value, "%ld", ValueType.via.i64 );
+                    printf( "Map value is int %" PRId64 "\n", ValueType.via.i64 );
+                    sprintf( mapdecodeReq->metadata->data_items[v].value, "%" PRId64, ValueType.via.i64 );
```
Comment (kraj) [inline src/wrp-c.c]: while this is a good change, do we also require c99 as minimum standard in CMake ?

Comment (schmidtw) [inline src/wrp-c.c]: It is there like this:
https://github.com/Comcast/wrp-c/blob/master/CMakeLists.txt#L21

Is that right or are we simply adding the declarations to configure the compiler to be c99?

Comment (kraj) [inline src/wrp-c.c]: this is fine however CMake way would be 
https://cmake.org/cmake/help/v3.1/prop_tgt/C_STANDARD.html

Comment (schmidtw) [inline src/wrp-c.c]: Thanks, added.

**Pull request merged**